springer-imis-series-migrationsgesellschaften.csl

```diff
@@ -0,0 +1,466 @@
+          </if>
+          <else>
+            <text term="letter" text-case="capitalize-first"/>
+          </else>
+        </choose>
+      </if>
+    </choose>
+    <names variable="recipient" delimiter=", ">
+      <label form="verb" prefix=" " suffix=" "/>
+      <name and="text" delimiter=", "/>
+    </names>
+  </macro>
+  <macro name="contributors">
+    <names variable="author">
+      <name and="text" name-as-sort-order="first" sort-separator=", " delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=", "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+    <text macro="recipient"/>
+  </macro>
+  <macro name="contributors-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter=", " initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="interviewer">
+    <names variable="interviewer" delimiter=", ">
+      <label form="verb" prefix=" " text-case="capitalize-first" suffix=" "/>
+      <name and="text" delimiter=", "/>
+    </names>
+  </macro>
+  <macro name="archive">
+    <group delimiter=". ">
+      <text variable="archive_location" text-case="capitalize-first"/>
+      <text variable="archive"/>
+      <text variable="archive-place"/>
+      <text variable="URL"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <group delimiter=". ">
+      <choose>
+        <if type="graphic report" match="all">
+          <text macro="archive"/>
+        </if>
+        <else-if type="article-magazine article-newspaper bill book chapter graphic legal_case legislation motion_picture paper-conference report song thesis" match="none">
+          <text macro="archive"/>
+        </else-if>
+      </choose>
+      <text variable="DOI" prefix="doi: "/>
+      <choose>
+        <if variable="DOI" match="all">
+          <choose>
+            <if variable="URL" match="any" type="webpage">
+              <choose>
+                <if type="legal_case" match="all">
+                  <text variable="URL"/>
+                </if>
+              </choose>
+              <group delimiter=" ">
+                <text term="accessed" text-case="capitalize-first"/>
+                <date variable="accessed" form="text" date-parts="year-month-day"/>
+              </group>
+            </if>
+          </choose>
+        </if>
+        <else-if variable="accessed">
+          <group delimiter=" ">
+          <text term="accessed" text-case="capitalize-first"/>
+                        <date variable="accessed" form="text" date-parts="year-month-day"/>
+          </group>
+        </else-if>
+      </choose>
+    </group>
+    <choose>
+      <if type="webpage" match="any">
+        <text variable="abstract"/>
```

Comment:
This is a bit puzzling -- what do you want the abstract for for webpages?

Reply:
@adam3smith thank you so much for your help! To be honest, we do not need abstracts for web pages, I can not explain why that's in the code. But the line can be deleted. 
Once again, many thanks for your support